Makefile

```diff
@@ -48,12 +42,6 @@ validation_requirements: piptools ## sync to requirements for testing & code qua
 doc_requirements: piptools
 	pip-sync -q requirements/doc.txt
 
-static: ## generate static files
-	python manage.py collectstatic --noinput
-
-shell: ## run Django shell
```

Comment:
Why does this no longer apply?

Reply:
I've never seen or heard of someone using it, and haven't seen it show up in docs. But it's still functional, so I guess I should leave it in in any case.